### PR TITLE
add e2e for v2 function directive

### PR DESCRIPTION
**Pull request by sundersc**
#### Description of changes
Add e2e test for v2 graphql function directive.

#### Description of how you validated changes
- E2E in local

#### Checklist
- [x] PR description included
- [ ] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/graphql-transformers-e2e-tests/src/__tests__/FunctionTransformerTestsV2.e2e.test.ts [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+import { LambdaHelper } from '../LambdaHelper';
+import { IAMHelper } from '../IAMHelper';
+
+jest.setTimeout(2000000);
+
+const cf = new CloudFormationClient('us-west-2');
+const customS3Client = new S3Client('us-west-2');
+const awsS3Client = new S3({ region: 'us-west-2' });
+const BUILD_TIMESTAMP = moment().format('YYYYMMDDHHmmss');
+const STACK_NAME = `FunctionTransformerTests-${BUILD_TIMESTAMP}`;
```
Comment (cjihrig) [inline packages/graphql-transformers-e2e-tests/src/__tests__/FunctionTransformerTestsV2.e2e.test.ts:22]: Can you add v2 in here and below as needed.

**Review by cjihrig — APPROVED** [review]
LGTM with a small comment

File: packages/graphql-transformers-e2e-tests/src/__tests__/FunctionTransformerTestsV2.e2e.test.ts (hunk 1 above)
Comment (sundersc) [reply packages/graphql-transformers-e2e-tests/src/__tests__/FunctionTransformerTestsV2.e2e.test.ts:22]: I will update.

Comment (sundersc) [reply packages/graphql-transformers-e2e-tests/src/__tests__/FunctionTransformerTestsV2.e2e.test.ts]: Updated.

**Pull request merged**